src/AsyncTCP.cpp

```diff
@@ -935,13 +956,12 @@ int8_t AsyncClient::_recv(tcp_pcb* pcb, pbuf* pb, int8_t err) {
             }
             if(!_ack_pcb) {
                 _rx_ack_len += b->len;
-            } else if(_pcb) {
-                _tcp_recved(_pcb, _closed_slot, b->len);
             }
-            pbuf_free(b);
         }
+        pbuf_free(b);
```

Comment:
The pbuf_free change is OK but the remaining changes no.
See here for more details: https://github.com/tbnobody/OpenDTU/discussions/2326#discussioncomment-10927299

This causes a drastic slowdown of the readings.